content/docs/2.10/scalers/prometheus.md

```diff
@@ -69,7 +69,10 @@ You can use `TriggerAuthentication` CRD to configure the authentication. It is p
 
 > 💡 **NOTE:**It's also possible to set the CA certificate regardless of the selected `authModes` (also without any authentication). This might be useful if you are using an enterprise CA.
 
-### Example
+**Auth for Azure managed service for Prometheus:**
+- See the `Azure managed service for Prometheus` section later on this page.
```

Comment:
We typically provide an overview here to help end-users understand what is required, samples are only a nice to have.

I think the unique fields for this feature is `azureManagedPrometheusResourceURL` and `cloud` if I'm not mistaken?

Reply:
done